activities/communication/community-call.md

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
It feels like this significantly duplicates /communication/running-community-call.md, and that they could be consolidated. Would you like me to try?

Reply:
The point here was that @Ainali mentioned these files are to support you whilst you are doing it and are meant to be the 1 file you follow. They are implementations with the right things and links in them.

I also think they can be simplified, because I think there still is quite a lot of background information in there that can be centralized whilst this file turns into a simple 'step list' or 'checklist' with all the relevant and right links in there.

I was trying to minimise destructiveness of this change. I think it might be better to fix this afterwards in a separate pull request, however if you think it needs to be solved now, feel free to take a stab.